### PR TITLE
Create arm64_ubuntu_16_04_LTS_android.json

### PR DESCRIPTION
**Pull request by AndrewSB**

### CODE REVIEW

**Comment by shahmishal** [comment]
@AndrewSB Email sent for additional information.

File: nodes/arm64_ubuntu_16_04_LTS_android.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  },
+  "node": {
+      "platform": "Android arm64",
+      "os_version": "Ubuntu 16.04 LTS"
+  },
+  "jobs": [
+    {
+      "display_name": "Swift - Android (Tools RA, Stdlib RD) (master)",
+      "branch": "master",
+      "preset": "buildbot_linux_crosscompile_android,tools=RA,stdlib=RD,build"
```
Comment (drodriguez) [inline nodes/arm64_ubuntu_16_04_LTS_android.json]: We probably want to change the display name and the preset. The Android ARM bot also has the same problem.

Even if the name and the preset name say tools are  Release+Assertions and stdlib is Release+DebugInfo, it seems to me that the preset is setting everything Release+Assertions.

The preset will have to change to the preset that compiles ARM64.

**Comment by Sun1016** [comment]
@Sun1016	Sun1016 started a review less than a minute ago Pending

**Comment by shahmishal** [comment]
Node: https://ci-external.swift.org/computer/ubuntu-16.04-android-arm64/
Job: https://ci-external.swift.org/job/oss-swift-RA-linux-ubuntu-16.04-android-arm64/

Thanks!

**Comment by drodriguez** [comment]
@shahmishal: it seems that the variable `armdir` is `/home/ubuntu/libiconv-libicu-android/armeabi-v7a` instead of `/home/ubuntu/libiconv-libicu-android/arm64-v8a`. It might have been copied from the configuration of the already existing Android node.

**Comment by drodriguez** [comment]
@shahmishal: Seems that someone (you?) changed the variable between the builds 3 and 5, but it is still incorrect: it is `/home/ubuntu/libiconv-libicu-android/armeabi-v8a` when it should be `/home/ubuntu/libiconv-libicu-android/arm64-v8a`. I have created a symbolic link between the wrong directory and the correct one, so it should work, but it will be good to have the right value in the variable anyway. Thanks for doing all this, by the way!

**Comment by shahmishal** [comment]
Sorry about that! 

I updated the job, please verify with this build https://ci-external.swift.org/job/oss-swift-RA-linux-ubuntu-16.04-android-arm64/14/console

**Pull request merged**